alicloud/table_alicloud_cs_kubernetes_cluster.go

```diff
@@ -504,6 +504,10 @@ func getCsKubernetesClusterARN(ctx context.Context, d *plugin.QueryData, h *plug
 //// TRANSFORM FUNCTIONS
 
 func csKubernetesClusterAkaTagsToMap(_ context.Context, d *transform.TransformData) (interface{}, error) {
+	if d.Value == nil {
```

Comment:
@ParthaI if we are already checking `d.Value == nil` then I don't think we need the check mentioned in lines https://github.com/turbot/steampipe-plugin-alicloud/pull/426/files#diff-01756e1a7a77698610d809a8875a0ab69232965568f82bf94296fa5560ad9ce1L508-L514.